framework/src/security/security_keymgr.c

```diff
@@ -114,7 +117,7 @@ int keymgr_get_key(security_handle hnd, security_key_type algo, const char *key_
 		SECAPI_FREE(pubkey_x);
 		SECAPI_RETURN(SECURITY_ALLOC_ERROR);
 	}
-	SECAPI_DATA_DCOPY(h_pubkey, pubkey_x);
+	SECAPI_PRIV_DCOPY(h_pubkey, pubkey_y);
```

Comment:
Is SECAPI_PRIV_DCOPY correct rather than SECAPI_DATA_DCOPY?

Reply:
they copy different member.